### DISCUSSION TITLE
Constant crashes

### DISCUSSION THREAD

**Discussion by huyz** [swiftbar/SwiftBar#376] [Q&A]
SwiftBar keeps dying on me and I'm wondering if this is happening to anyone else, as I can't get any search hits to that effect.

SwiftBar v1.4.4
macOS 13.3.1(a) (Ventura, ARM)

**Comment by melonamin** [comment]
Hey @huyz,

didn't hear about this from anyone yet. Any details you're willing to share? Plugins you're running, console.app logs,  etc.

**Reply by huyz** [reply]
I'll add my list of crashes to https://github.com/swiftbar/SwiftBar/issues/381
